javascript/modules/client/src/client-addressList/internal/encoding.ts

```diff
@@ -0,0 +1,106 @@
+import { WhitelistVoting__factory } from "@aragon/core-contracts-ethers";
```

Comment:
Why do we have `<plugin>/internal/encoding.ts` as well as `<plugin>/internal/client/encoding.ts`?  (applies to the rest)